### ISSUE TITLE
Add Information to TM description

### ISSUE THREAD

**Issue by szymczdm** [JacquesCarette/Drasil#521]
When updating the TMs (on @halonazhao's branch) the descriptions have changed, but some of that information is still useful. We need to be able to re-introduce the lost information (one example is pictured here):

![tms](https://user-images.githubusercontent.com/235140/39977979-abdaf53c-570b-11e8-8384-2e4ce795a6f2.PNG)
Left is build, Right is stable

**Comment by szymczdm** [comment]
Just realized this is one part of #507

**Comment by smiths** [comment]
The missing information is usually related to the derivation.  We have looked at derivation as a section that follows the table, but this is really just a shorter version of the same kind of information.

**Comment by smiths** [comment]
This issue is tied to #523.  When I go through the comparison of manual versus Drasil, I'll check this issue, along with the others.

**Comment by smiths** [comment]
Yes, we can close this issue.  The issue relates to adding the Notes field, which has been done.  I also verified that the specific motivating example used in this issue has been fixed.  There are some associated issues that have been linked to this one.  These other issues would also be solved by the Notes field, but I need to verify that we actually moved the right information to the Notes field for these other issues.  (We sometimes miss some details when updating the Notes field.)  I'll close this issue and follow up on the others now.
